data/migrations/23.lua

```diff
@@ -1,3 +1,5 @@
 function onUpdateDatabase()
-	return false
+	print("> Updating database to version 24 (Add player direction)")
+	db.query("ALTER TABLE `players` ADD `direction` int(1) unsigned NOT NULL DEFAULT 2")
```

Comment:
`DEFAULT 2` is in this case `DIRECTION_SOUTH` which is default value for creatures in https://github.com/otland/forgottenserver/blob/master/src/creature.h#L512

Reply:
I would add it right next to outfit data.

Reply:
I'm against using variables in migrations. We don't expect this one to change but in case it did, it would be quite bad.

Reply:
I've moved `player->direction` closer outfit data.

Reply:
@ranisalt How about that migration? It's ok to have numeric value as @DSpeichert said or should I change to `DIRECTION_SOUTH`?

Reply:
I agree with @DSpeichert